sky/packages/sky/lib/src/widgets/editable_text.dart

```diff
@@ -203,21 +204,22 @@ class EditableText extends StatefulComponent {
     else if (!focused && _cursorTimer != null)
       _stopCursorTimer();
 
-    if (!value.composing.isValid) {
+    Widget text;
+    if (value.composing.isValid) {
+      TextStyle composingStyle = style.merge(const TextStyle(decoration: underline));
+      text = new StyledText(elements: [
+        style,
+        value.textBefore(value.composing),
+        [composingStyle, value.textInside(value.composing)],
+        value.textAfter(value.composing)
+      ]);
+    } else {
       // TODO(eseidel): This is the wrong height if empty!
```

Comment:
If I understand correctly, this refers to the fact that the Text widget will compute a size of (0, 0) if the text is empty.  That hasn't changed.

When the text is empty, the size of the overall EditableText should now match the size of the cursor, which is derived from the font size.